docs/using-kontena/volumes.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Volume Management
+---
+
+# Volumes
+
+Kontena 1.2 provides **experimental** support for managing persistent service data using Docker volumes. The exact details of how these Kontena volumes are managed may still change as the implementation evolves.
+
+Volumes are supported as first-class objects in the Kontena Master API, which can be referred to from stack YAML definitions.
+
+Kontena tracks the available Docker volume drivers on each host node, as shown in kontena node show, and will only deploy services using such a volume driver onto a host node that provides that volume driver. In order to use any other volume drivers than the default local driver, those volume drivers must be provided by Docker plugins on the host nodes. Kontena itself does not yet support provisioning Docker plugins onto the grid's host nodes.
+
+
+## Managing volumes with CLI
+
+Volumes can be managed from the Kontena CLI using the `kontena volume xyz` commands.
+
+### Creating volumes
+
+`kontena volume create --driver rexray --scope instance my-volume`
+
+See volume [scopes](#volume-scopes) for details on the different scopes.
+
+### Listing volumes
+
+`kontena volume ls`
+
+### Deleting volumes
+
+`kontena volume rm my-volume`
+
+Deleting a volume that is still in use by any Kontena services will fail. Deleting an unused volume will remove the underlying Docker volumes from the host nodes.
```

Comment:
Depending on the volume driver, this may also remove the actual backing storage for the volume.